automates/text_reading/src/main/resources/org/clulab/aske_automates/grammars/vars.yml

```diff
@@ -26,4 +26,16 @@ passive_mark_verbs: "marked|defined|indicated|characterized"
 
 transparent_nouns: "level|amount|quantit" # do we need these in general?
 
-# Todo: nmod_from "X will follow from Y"
\ No newline at end of file
+# Todo: nmod_from "X will follow from Y"
+
+# FUNCTION TRIGGERS
+
+functionTrigger1: "[lemma = influence]|[lemma = modify]|[lemma = govern]|[lemma = limit]|[lemma = control]|[lemma = dominate]"
```

Comment:
@maxaalexeeva, I have some questions regarding this comment. Are the triggers automatically translated as lemmas even if I omit `lemma=` part? Also, if I want to keep some triggers as words rather than lemmas, I should keep them as `word=` forms. Is that correct?

Reply:
Good questions. I think the lemma part can go inside the rule; I forget the exact syntax but something like `trigger = [lemma=/${trigger_group1}/]`

You can do words instead of lemmas, yes. Can you give me an example for when you need that? Basically, everything you have inside slashes /.../ is a regex. You can have triggers in the vars file be parts of words and then in the rule, use ^ to indicate that the trigger should start with a certain letter sequence, eg (in vars.yml) `triggers_group1:"defin|influence"` can be used to capture both verbs and gerunds in the trigger definition with `trigger = [lemma=/^${trigger_group1}/]` 

We can discuss more in the meeting next week or stay for a few minutes after the TR meeting this Thursday.

Reply:
Thanks for the clarification! I wanted some trigger groups (i.e. functionTrigger2 or 4) to be words than lemmas to exclude their verbal usages (if they have). I'll try to fine-tune the triggers by using regex. Thank you for the great advices :)

Reply:
@alicekwak, another related thing: you can add allowable parts of speech as well, e.g., if you want to only have base form of verbs, you can say something like `trigger=[lemma=/<whatever verbs you want>/ & tag = "VB"]` or if you want anything verb-y, you can do `tag = /^VB/` (tag name starts with "VB"). I think right now these part of speech (POS) tags are what we use in our system: https://www.ling.upenn.edu/courses/Fall_2003/ling001/penn_treebank_pos.html